app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    User.limit(1)
```

Comment:
- [ ] the current_user should reflect not limit the number of users.

Reply:
Hey, @dasileker I don't understand what you mean by should reflect, could you describe more?